man/fi_getinfo.3.md

```diff
@@ -514,6 +520,14 @@ fabric.  See `fi_av`(3).
 : Address is a Cray proprietary format that is used with their GNI
   protocol.
 
+*FI_ADDR_STR*
+: Address is a formatted character string.  The length and content of
+  the string is address and/or provider specific, but follows this model:
+
+  address_family[;[node][;[service][;[field3]...]]]
+
+  Examples: AF_INET;10.31.6.12;7471, AF_INET;;7471
```

Comment:
Do we want to allow specifying the delimiter?  Or add quotes around the parts?  E.g. "AF_INET","10.0.0.1"